src/common/jsonapi.c

```diff
@@ -21,10 +21,39 @@
 #include "mb/pg_wchar.h"
 #include "port/pg_lfind.h"
 
-#ifndef FRONTEND
+#ifdef FRONTEND
+#include "pqexpbuffer.h"
+#else
+#include "lib/stringinfo.h"
 #include "miscadmin.h"
```

Comment:
Same question. Are changes to this file intentional? Not clear why we need to change json manipulation for our change here

Reply:
SASL exchange between client and server(for sending discovery_uri and scope) uses json object. Few changes are required for now( since it is on master branch). Will need to figure out the optimal changes.